README.md

```diff
@@ -10,7 +10,7 @@
 |           |   C                              |             J                    |
 |-----------|----------------------------------|----------------------------------|
 | Released  | [icu4c 71.1](./apidoc/released/icu4c) | [icu4j 71.1](./apidoc/released/icu4j) |
-| Dev       | [icu4c 71rc](./apidoc/dev/icu4c)      | [icu4j 71rc](./apidoc/dev/icu4j)      |
+| Dev       | [icu4c 72rc](./apidoc/dev/icu4c)      | [icu4j 71rc](./apidoc/dev/icu4j)      |
```

Comment:
Note that you are modifying the same line in both PRs. After you merge one, you will need to resolve the conflict in the other.